internal/graphql/schema.graphql

```diff
@@ -70,4 +74,11 @@ enum Driver  {
   MYSQL
   POSTGRESQL
   SQLITE
+}
+
+enum ArchiveFormat {
+  """
+  base64 encoded tar format.
+  """
+  B64_TAR
```

Comment:
Do we need a one type enum? We can just pass a static string here, right?

Reply:
I prefer this, using the generated code looks nicer IMO, in addition, I want to keep this aligned with the server side GQL schema.